.circleci/config.yml

```diff
@@ -91,6 +145,7 @@ workflows:
           filters:
             branches:
               only: master
+      # Run on tag pushes
```

Comment:
This makes me think we should automate android release deployment to bintray as well. Since we can do that now unlike earlier with SonaType!

Reply:
I agree! Sounds like a good idea for another PR.

Reply:
If its not a big change then could be done part of the same PR, but I am fine either way.